From 32d5b6208c42152149aeca838b57ba5afc3a9a36 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Wed, 25 Jul 2018 18:59:20 +0000
Subject: [PATCH 315/462] Mantis 0051292: ConveyorDev: integrate bhs server
 into platform build

added definitions for JarBhsServer, JarHTTPCoreNio, JarSpongyCastleCore, JarSpongyCastleProv, JarSpongyCastlePkix, JarBouncyCastleProv, JarBouncyCastlePkix,

git-svn-id: svn://localhost/SelfServiceBuild/trunk@545 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/java/AntProperties.xml | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/SelfServiceBuild/Build/java/AntProperties.xml b/SelfServiceBuild/Build/java/AntProperties.xml
index 4f0e623..3758f82 100644
--- a/SelfServiceBuild/Build/java/AntProperties.xml
+++ b/SelfServiceBuild/Build/java/AntProperties.xml
@@ -162,6 +162,7 @@
 <property name="JarHTTPFluent"              value="${DIR_GLOBALS}/ThirdParty/httpClient/4.2.3/lib/fluent-hc-4.2.3.jar"/>
 <property name="JarHTTPCache"               value="${DIR_GLOBALS}/ThirdParty/httpClient/4.2.3/lib/httpclient-cache-4.2.3.jar"/>
 <property name="JarHTTPMime"                value="${DIR_GLOBALS}/ThirdParty/httpClient/4.2.3/lib/httpmime-4.2.3.jar"/>
+<property name="JarHTTPCoreNio"             value="${DIR_GLOBALS}/ThirdParty/httpCore/4.2.2/lib/httpcore-nio-4.2.2.jar"/>
 
 <!-- SVNKit -->
 <property name="JarSVNKit"                  value="${DIR_GLOBALS}/ThirdParty/svnkit/1.7.5/lib/svnkit-1.7.5-v1.jar"/>
@@ -267,6 +268,7 @@
 <property name="JarMORPlugin-HamIdCheckIDrop" value="${JarDir}/mips-morplugin-HamIdCheckIDrop.jar"/>
 <property name="JarIDropIdCheck"              value="${JarDir}/idcheck.jar"/>
 <property name="JarExtractBilling"            value="${JarDir}/extractBilling.jar"/>
+<property name="JarBhsServer"                 value="${JarDir}/BhsServer.jar"/>
 
 <!-- slf4j -->
 <property name="JarSlf4jApi"                value="${DIR_GLOBALS}/ThirdParty/slf4j/1.7.21/slf4j-api-1.7.21.jar"/>
@@ -324,6 +326,14 @@
 <property name="JarNettyAll"                value="${DIR_GLOBALS}/ThirdParty/netty/4.1.1/netty-all-4.1.1.Final.jar"/>
 
 
+<!-- Castle -->
+<property name="JarSpongyCastleCore"        value="${DIR_GLOBALS}/ThirdParty/spongyCastle/1.52.0.0/sc-core-1.52.0.0.jar"/>
+<property name="JarSpongyCastleProv"        value="${DIR_GLOBALS}/ThirdParty/spongyCastle/1.52.0.0/scprov-1.52.0.0.jar"/>
+<property name="JarSpongyCastlePkix"        value="${DIR_GLOBALS}/ThirdParty/spongyCastle/1.52.0.0/scpkix-1.52.0.0.jar"/>
+<property name="JarBouncyCastleProv"        value="${DIR_GLOBALS}/ThirdParty/bouncyCastle/1.5.2/bcprov-jdk15on-152.jar"/>
+<property name="JarBouncyCastlePkix"        value="${DIR_GLOBALS}/ThirdParty/spongyCastle/1.5.2/bcpkix-jdk15on-152.jar"/>
+
+
 <!-- apache common -->
 <property name="JarCommonsIo"               value="${DIR_GLOBALS}/ThirdParty/commons-io/1.4/commons-io-1.4.jar"/>
 <property name="JarCommonsConfiguration"    value="${DIR_GLOBALS}/ThirdParty/commons-configuration/1.6/commons-configuration-1.6.jar"/>
-- 
2.41.0.windows.1

